frontend/src/components/Navbar/index.tsx

```diff
@@ -88,6 +147,29 @@ export default function Navbar() {
             >
               Hi, {user?.name}!
             </Typography>
+
+            <Menu
+              id="basic-menu"
```

Comment:
@prajjwalkapoor For mobile screens, I think it would be much better if we display only Avatar  in Profile-container. By clicking the Avatar,  Menu bar will open. This will help optimize the limited screen space on mobile devices. Let me know your thoughts on this.

Reply:
Nice idea, let's go this.

Reply:
I had successfully implemented the design of ProfileContainer as discussed for Mobile devices and set the menu width according to the length of the username making it nearly the same as the button width. Please review it and let me know if any changes are required.

Reply:
Awesome @Pratik2026, You did a great job on this.